Frontend/ui-customer/src/Screens/CustomerUtils/RideSummaryScreen/ComponentConfig.purs

```diff
@@ -0,0 +1,275 @@
+    sourceToDestinationConfig' =
+      config
+        { 
+          id = Just "rideScreenPickupMap",
+          separatorMargin = 0,
+          showDestination = tripCategory.tag /= CTA.Rental,
+          sourceImageConfig
+        { imageUrl = fetchImage FF_COMMON_ASSET "ny_ic_source_dot"
+        , margin = MarginTop 3
+        , width = V 18
+        , height = V 18
+        }
+      , sourceTextConfig
+        { text = if tripCategory.tag == CTA.InterCity then sourceCity else sourceArea
+        , padding = Padding 2 0 2 2
+        , margin = MarginHorizontal 12 15
+        , color = Color.black900
+        , ellipsize = true
+        , maxLines = 1
+        , textStyle = FontStyle.SubHeading2
+        }
+      , rideStartedAtConfig
+        { text = sourceFullAddress
+        , color = Color.black600
+        , visibility = VISIBLE
+        , padding = Padding 2 0 2 2
+        , margin = MarginHorizontal 12 17
+        , maxLines = 2
+        , ellipsize = true
+        , textStyle = FontStyle.ParagraphText
+        }
+      , rideEndedAtConfig
+        { text = destinationFullAddress
+        , color = Color.black600
+        , visibility = VISIBLE
+        , padding = Padding 2 0 2 2
+        , margin = MarginHorizontal 12 15
+        , maxLines = 2
+        , ellipsize = true
+        , textStyle = FontStyle.ParagraphText
+        }
+      , destinationImageConfig
+        { imageUrl = fetchImage FF_COMMON_ASSET "ny_ic_destination"
+        , margin = MarginTop 0
+        , width = V 20
+        , height = V 23
+        }
+      , destinationTextConfig
+        { text = if tripCategory.tag == CTA.InterCity then destinationCity else destinationArea
+        , padding = Padding 2 0 2 2
+        , margin = MarginHorizontal 12 15
+        , color = Color.black900
+        , ellipsize = true
+        , maxLines = 1
+        , textStyle = FontStyle.SubHeading2
+        }
+      , horizontalSeperatorConfig
+        { visibility = GONE
+        , background = Color.grey900
+        , padding = Padding 2 0 2 2
+        , margin = Margin 12 12 15 0
+        }
+      , pillsConfig
+        { visibility = GONE,
+          pillList = []
+        }
+      , overrideSeparatorCount = if sourceFullAddress == "" then 4 else 7
+      }
+  in
+    sourceToDestinationConfig'
+
+      
+rideSummaryCardConfig :: BookingAPIEntity -> RideSummaryCard.Config
+rideSummaryCardConfig (BookingAPIEntity entity) = 
+  let 
+    vehicleServiceTier = entity.vehicleServiceTier
+    currency = entity.currency
+    estimatedFare = entity.estimatedFare
+    isAirConditioned = (fromMaybe false entity.isAirConditioned) || (fromMaybe false (map (\x -> x >= 0.0) entity.vehicleServiceTierAirConditioned)) 
+    vehicleServiceTierAirConditioned = isAirConditioned || (case vehicleServiceTier of --TODO :  need this to be configured from backend 
+                                                                    "AUTO_RICKSHAW" -> false
```

Comment:
create two arrays one for true value and one for false values and check if the vehicle exists in those arrays or not.